src/scss/root/_index.scss

```diff
@@ -57,45 +125,13 @@ $tokens: () !default;
 	}
 }
 
-@each $breakpoint, $breakpointValue in $breakpoints {
-	@media only screen and ($breakpointValue) {
-		:root {
-			// Alias Tokens
-			@each $name, $value in map.get($tokens, $breakpoint, 'alias') {
-				--#{$name}: #{$value};
-			}
-			// Component Tokens
-			@each $componentName, $componentItem in map.get($tokens, $breakpoint, 'components') {
-				@if list.length($componentItem) == 0 {
-					@warn 'Component "#{$componentName}" is configured with no options';
-				} @else {
-					@if type-of($componentItem) == map {
-						@each $componentProperty, $componentValue in $componentItem {
-							@if type-of($componentValue) != map {
-								--c-#{$componentName}-#{$componentProperty}: #{$componentValue};
-							}
-						}
-					}
-				}
-			}
-			// Block Tokens
-			@each $blockName, $blockItem in map.get($tokens, $breakpoint, 'blocks') {
-				@if list.length($blockItem) == 0 {
-					@warn 'Block "#{$blockName}" is configured with no options';
-				} @else {
-					@if type-of($blockItem) == map {
-						@each $bProperty, $bValue in $blockItem {
-							@if type-of($bValue) != map {
-								--b-#{$blockName}-#{$bProperty}: #{$bValue};
-							}
-						}
-					}
-				}
-			}
+@if list.length($tokens) != 0 {
+	@each $breakpoint, $breakpointValue in $breakpoints {
+		@media only screen and ($breakpointValue) {
```

Comment:
only `screen` and not `print` as well?

Reply:
Can add print too if you think that's worthwhile? or could just use `all`?

Reply:
can use `all`.  I think there's only three options... `screen`, `all`, or `print`.

Reply:
or I think you can leave it off as well.

Reply:
> Media types describe the general category of a device. Except when using the not or only logical operators, the media type is optional and the all type is implied.

Reply:
Removed for now - we can revisit in the future if needed